website/static/js/addProjectPlugin.js

```diff
@@ -144,6 +153,54 @@ var AddProject = {
                                 name : 'projectName'
                             })
                         ]),
+                        window.contextVars.currentUser.institutions.length ? m('.form-group.m-v-sm', [
+                            m('label.f-w-lg.text-bigger', 'Affiliation'),
+                            m('a', {onclick: function(){
+                                $('.img-circle.fa.fa-check').addClass('hidden');
+                                ctrl.newProjectInstitutions = [];
+                            }, style: {float: 'right'}},'Remove all'),
+                            m('a', {onclick: function(){
+                                $('.img-circle.fa.fa-check').removeClass('hidden');
+                                ctrl.newProjectInstitutions = window.contextVars.currentUser.institutions.map(
+                                    function(inst){return inst.id;}
+                                );
+                            }, style: {float: 'right', marginRight: '5px'}}, 'Select all'),
+                            m('table', m('tr', window.contextVars.currentUser.institutions.map(
+                                function(inst){
+                                    return m('td',
+                                        m('a', {onclick: function(){
+                                            $('#instLogo' + inst.id).toggleClass('hidden');
+                                            if (ctrl.newProjectInstitutions.indexOf(inst.id) !== -1){
+                                                ctrl.newProjectInstitutions.pop(inst.id);
+                                            } else {
+                                                ctrl.newProjectInstitutions.push(inst.id);
+                                            }
+                                        }},m('', {style: {position: 'relative',  margin: '10px'}, width: '45px', height: '45px'},
+                                            [
+                                            m('img.img-circle.text-muted',
+                                                {
+                                                    src: inst.logo_path, width: '45px', height: '45px',
+                                                }
+                                            ),
+                                            m('i.img-circle.fa.fa-check#instLogo' + inst.id,
+                                                {
+                                                    style: {
+                                                        color: 'lightgreen',
```

Comment:
Use the osf-style color palette whenever possible. https://centerforopenscience.github.io/osf-style/style.html#colors

How about #C7FFC7?